### ISSUE TITLE
add output_shape to tf.layers.conv2d_transpose

### ISSUE THREAD

**Issue by Burton2000** [tensorflow/tensorflow#19236]
### System information
- **Have I written custom code (N/A)**:
- **OS Platform and Distribution (Windows 10)**:
- **TensorFlow installed from (binary)**:
- **TensorFlow version (1.8.0)**:
- **Python version 3.6.1**: 
- **Bazel version (N/A)**:
- **GCC/Compiler version (N/A)**:
- **CUDA/cuDNN version** N/A:
- **GPU model and memory** N/A:
- **Exact command to reproduce** N/A:

### Describe the problem
Currently tf.layers.conv2d_transpose doesn't have an output_shape parameter like tf.nn.conv2d_transpose does. As a result when using strides > 1 the output shape is not defined 

https://github.com/tensorflow/tensorflow/issues/2118#issuecomment-215488127

Would it be possible to have this added in to the layers api so we can specify output shape.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by Burton2000** [comment]
Yes

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
